core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -944,23 +969,28 @@ class PlaintextConsumerTest extends BaseConsumerTest {
   @Test
   def testMultiConsumerDefaultAssignment() {
     // use consumers and topics defined in this class + one more topic
-    sendRecords(100, tp)
-    sendRecords(100, tp2)
+    val producer = createProducer()
+    sendRecords(producer, numRecords = 100, tp)
+    sendRecords(producer, numRecords = 100, tp2)
     val topic1 = "topic1"
-    val subscriptions = Set(tp, tp2) ++ createTopicAndSendRecords(topic1, 5, 100)
+    val subscriptions = Set(tp, tp2) ++ createTopicAndSendRecords(producer, topic1, 5, 100)
 
     // subscribe all consumers to all topics and validate the assignment
-    val consumerPollers = subscribeConsumers(consumers, List(topic, topic1))
 
+    val consumerGroup = Buffer[KafkaConsumer[Array[Byte], Array[Byte]]]()
```

Comment:
Since `consumerGroup` refers to the logical group, perhaps `consumersInGroup` or something along those lines would be less confusing?